app/Models/Foundation/Summit/Factories/SummitEventFactory.php

```diff
@@ -137,6 +137,14 @@ static public function populate(Summit $summit, SummitEvent $event, array $paylo
             $event->setSubmissionSource(trim($payload['submission_source']));
         }
 
+        if(isset($payload['overflow_streaming_url'])) {
+            $event->setOverflowStreamingUrl(trim($payload['overflow_streaming_url']));
```

Comment:
this is not correct please review the use

Reply:
done